### PR TITLE
TS-3 remove mock db

### PR DESCRIPTION
**Pull request by bbenetskyy**
### This PR will wait for Review not Longer than 24h.

Here was removed fake DB. Correctly Registered Azure Data Service **For Android Device Family Only**

### CODE REVIEW

File: BagiraTrainings/BagiraTrainings/BagiraTrainings/ViewModels/BaseViewModel.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ namespace BagiraTrainings.ViewModels
     public class BaseViewModel<TModel> : INotifyPropertyChanged
         where TModel : IHaveId, new()
     {
-        public IDataStore<TModel> DataStore => DependencyService.Get<IDataStore<TModel>>() ?? new MockDataStore<TModel>();
+        public IDataStore<TModel> DataStore => DependencyService.Get<IDataStore<TModel>>();
```
Comment (itsurkan) [inline BagiraTrainings/BagiraTrainings/BagiraTrainings/ViewModels/BaseViewModel.cs]: IDataStore це Singleton?

File: BagiraTrainings/BagiraTrainings/BagiraTrainings/Services/AzureDataStore.cs [hunk 2]
```diff
@@ -12,11 +12,17 @@ public class AzureDataStore<T>: IDataStore<T>
         where T : IHaveId, new()
     {
         public IMobileServiceClient MobileServiceClient { get; private set; }
-        public IMobileServiceTable<T> Table { get; private set; } 
+        public IMobileServiceTable<T> Table { get; private set; }
+
+        public AzureDataStore()
+            :this(new MobileServiceClient("https://bagiratrainings.azurewebsites.net"))
```
Comment (itsurkan) [inline BagiraTrainings/BagiraTrainings/BagiraTrainings/Services/AzureDataStore.cs]: Думаю сюди теж варто додати DI IMobileServiceClient

Comment (bbenetskyy) [reply BagiraTrainings/BagiraTrainings/BagiraTrainings/Services/AzureDataStore.cs]: Ok will be fixed, but in English next time. 😉

**Comment by bbenetskyy** [comment]
Time is passed, PR will be merged now.

**Pull request merged**